client/my-sites/site-settings/form-security.jsx

```diff
@@ -119,6 +125,14 @@ const connectComponent = connect( ( state ) => {
 		'akismet'
 	);
 
+	const hasAkismetFeature = siteHasFeature( state, siteId, WPCOM_FEATURES_AKISMET );
+	const hasAntiSpamFeature = siteHasFeature( state, siteId, WPCOM_FEATURES_ANTISPAM );
+	const hasJetpackAntiSpamProduct =
+		getSiteProducts( state, siteId )?.filter( isJetpackAntiSpam ).length > 0;
```

Comment:
Here `.some( isJetpackAntiSpam )` is a bit nicer 🙂

Reply:
Thanks for the input. I adjusted accordingly.